src/test_mail_room.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+import pytest
+
+
+STATE4_TABLE = [
+    (10, 10),
+    (11, 11),
+    # ('0', SystemExit),
+]
+# def test_mailroom():
+    # from mailroom_functions import mailroom_functions
+
+
+def test_mailroom_initialize_donor_dict():
+    from mailroom_functions import initialize_donor_dict
+    assert type(initialize_donor_dict()) == dict
```

Comment:
you might want to add tests that assert that the stuff in the dictionary looks like what you expect.  In other words,what type of value are the keys expected to be?  The values? Check to be sure that you are in fact building what you think you are.